### PR TITLE
Jia/987 app download rendering

### PR DESCRIPTION
**Pull request by jia-deriv**
image

### CODE REVIEW

File: libs/templates/src/lib/trade/app-download/index.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+  return (
+    <Card.ContentRight
+      color="gray"
+      align="start"
+      size="lg"
+      content={content}
+      className="items-end lg:gap-gap-lg"
+      header="title here"
+      // description="description here"
```
Comment (Nuzhy-Deriv) [inline libs/templates/src/lib/trade/app-download/index.tsx:17]: delete these 2 lines

File: libs/templates/src/lib/trade/app-download/index.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    >
+      <div className="flex flex-col gap-gap-xl lg:gap-gap-2xl">
```
Comment (Nuzhy-Deriv) [inline libs/templates/src/lib/trade/app-download/index.tsx:19]: semantic will handle this.
```suggestion
      <div className="flex flex-col gap-gap-2xl">
```

File: libs/templates/src/lib/trade/app-download/index.tsx (hunk 1 above)
Comment (jia-deriv) [reply libs/templates/src/lib/trade/app-download/index.tsx]: done

File: libs/templates/src/lib/trade/app-download/index.tsx (hunk 2 above)
Comment (jia-deriv) [reply libs/templates/src/lib/trade/app-download/index.tsx]: done changes

File: libs/templates/src/lib/trade/app-download/data.tsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        height={536}
+        alt="MT5 App"
+      />
+      <OptimizedImage
+        className="block lg:hidden"
+        imageName="trade/features/hk_trade_feature_mobile_mt5.png"
+        width={809}
+        height={460}
+        alt="MT5 App"
+      />
```
Comment (prince-deriv) [inline libs/templates/src/lib/trade/app-download/data.tsx:28]: `OptimizedImage` component is already handling the proper rendering of image. We don't need to create this manually from our side.

Comment (jia-deriv) [reply libs/templates/src/lib/trade/app-download/data.tsx]: Done - use one image instead of two different size

**Review by Nuzhy-Deriv — DISMISSED** [review]

**Review by mohsen-deriv — DISMISSED** [review]

**Review by Nuzhy-Deriv — APPROVED** [review]

**Pull request merged**